src/python/pants/backend/python/subsystems/python_setup.py

```diff
@@ -80,8 +80,7 @@ def register_options(cls, register):
 
   @property
   def interpreter_constraints(self):
-    return (self.get_options().interpreter_constraints or self.get_options().interpreter or
-            [self.get_options().interpreter_requirement or b''])
+    return tuple(self.get_options().interpreter_constraints)
```

Comment:
So was the previous version simply broken? There is no `--interpreter-requirement` option.

Reply:
And since you're here and fixing this, can you also delete the stale comment on line 22 above?

Reply:
I believe so, yes. Guessing it was left behind after a deprecation.